src/main/resources/static/swagger.yaml

```diff
@@ -43,6 +43,70 @@ paths:
                   id:
                     type: string
                     format: uuid
+  /sessions:
+    post:
+      tags:
+        - frontend
+      summary: Login user
+      description: Create a session for user / doctors.
+      operationId: createSession
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SessionRequest'
+      responses:
+        '200':
+          description: Request created in the consent manager
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  accessToken:
+                    type: string
+                    example: bearer yJhbGciOiJSUzI1Ni.IsInR5cCIgOiAiSldUIiwia2lkIiA6ICJrVVp.2MXJQMjRyYXN1UW9wU2lWbkdZQUZIVFowYVZGVWpYNXFLMnNibTk0In0
+        '401':
+          description: Invalid username or password
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+  /users:
+    post:
+      tags:
+        - frontend
+      summary: Create user
+      description: Create HIU user who can access HIU application
+      parameters:
+        - $ref: '#/components/parameters/authorization'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UserCreationRequest'
+      responses:
+        '200':
+          description: Successful creation of consent artefact.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PatientSearchRepresentation'
```

Comment:
this ought to return user representation, right?

Reply:
We don't return anything, as it's an API we created for creating new doctor login.